### ISSUE TITLE
Implement and APPS import / export backend view

### ISSUE THREAD

**Issue by micker** [FLEXIcontent/flexicontent-cck#31]
Information/data exported imported:
- (SQL data) type (and its configuration properties/parameters)
- (SQL data) type's assigned fields and configuration of these fields
- (SQL data) type's template configuration (fields assigned to item layout)
- (files) type 's template files (if not-built-in)

**Comment by ggppdk** [comment]
Work started in v3.0 (see export in field configuration form) we will complete it in v3.1

**Comment by micker** [comment]
Great thé goal is to export import type (type +field+template) at end ?

**Comment by ggppdk** [comment]
Yes, we already have code for doing all of these (so we can reuse it), 
we have already have code for:
- exporting CSV/SQL table records
- importing CSV/SQL table records
- creating a ZIP archive with installer XML file

just need to reuse it, and solve some implementations issue, e.g. the export contains an ID but we probably want ZERO id on import or "selecting" ... ID on import
